data/pages.json

```diff
@@ -103,5 +103,10 @@
         "description": "Imposter 404",
         "imageUrl": "https://raw.githubusercontent.com/Rohanfizz/404-templates/main/__screenshots/imposter.png?raw=true",
         "url": "imposter/404.html"
-    }
+    },
+    {
+        "description": "Desert 404",
+        "imageUrl": "/__screenshots/desert.png?raw=true",
```

Comment:
The imageUrl format must be like all the other ones

Reply:
It's correct that the image is not visible locally, it's not in the repository yet. It will work when pushed.

Reply:
The URL is just a string, I don't think that changing the value is breaking the index.html, maybe there's another error.

Reply:
I resolved the merge conflict as well, so now it shall work